src/js/jsx/sections/style/EffectsPanel.jsx

```diff
@@ -186,6 +187,8 @@ define(function (require, exports, module) {
             dialog.toggle(event);
             this._forceVisible();
             this.getFlux().actions.layerEffects.addEffect(this.props.document, layers, effectType);
+
+            headlights.logEvent("effect", "create", effectType);
```

Comment:
The effect type here is an exception to my wish above to have all the categories and events be lower-case and dash-separated. Whatever effect type is can be jammed into the name field here as long as it's a string.